package-lock.json

```diff
@@ -1,16551 +1,8 @@
 {
   "name": "answers-hitchhiker-theme",
   "version": "1.30.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
can we switch back to lockfileVersion 2?

Reply:
that comes from npm 7 and is backwards compatible with version 1. https://github.blog/2021-02-02-npm-7-is-now-generally-available/#changes-to-the-lockfile

Reply:
Ah I had this backward, switched back